FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/HealthCheckBackgroundService.cs

```diff
@@ -68,7 +68,7 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
                     List<Task> listenerTasks = _healthCheckListeners.Select(l => l.ProcessHealthStatusAsync(healthStatus, cancellationToken)).ToList();
                     foreach (HealthCheckResult component in healthStatus.HealthCheckResults)
                     {
-                        _metricsLogger.LogHealthStatusMetric(component.Name, !component.IsCritical, component.Status == Models.HealthCheckStatus.HEALTHY ? 1 : 0);
+                        _metricsLogger.LogHealthStatusMetric(component.Name, !component.IsCritical, component.Status == HealthCheckStatus.HEALTHY ? 1 : 0);
```

Comment:
is it possible to use the status enum value directly here?

Reply:
is it possible to use the status enum value directly here?

Reply:
Updated it here, thanks.